### PR TITLE
Reload realtime after reload queue is created

### PR DESCRIPTION
**Pull request by pbench**

### CODE REVIEW

File: server/src/data_worker.rs [hunk 1]
```diff
@@ -286,13 +281,19 @@ impl DataWorker {
...
-        }
-
         let mut real_time_messages_consumer = self.connect_real_time_queue(channel).await?;
         let mut reload_consumer = self.connect_reload_queue(channel).await?;
 
+        // We perform the realtime reload *after* the real_time queue is created, otherwise we may loose some
+        // realtime messages.
+        //
+        // We may be disconnected from rabbitmq after the initial realtime reload was done.
+        // In this case, we don't want to perform a the realtime reload again.
```
Comment (woshilapin) [inline server/src/data_worker.rs:291]:
```suggestion
        // In this case, we don't want to perform a realtime reload again.
```

File: server/src/data_worker.rs [hunk 2]
```diff
@@ -274,7 +267,9 @@ impl DataWorker {
             // listen for Reload order
             let has_reload_message = reload_consumer.next().await;
             info!("Received a message on the reload queue.");
-            self.handle_reload_message(has_reload_message, channel)
+            // do not reload realtime here, because the realtime queue is not created yet, so we may
+            // loose some realtime messages
```
Comment (woshilapin) [inline server/src/data_worker.rs:271]: Not sure: shouldn't this be `lose`? (https://www.merriam-webster.com/words-at-play/lose-vs-loose-usage)

```suggestion
            // lose some realtime messages
```

**Review by woshilapin — APPROVED** [review]
I've seen the comment, but still, I'm not sure that I completely understand the purpose of `initial_realtime_reload_done`.

**Comment by pbench** [comment]
> I've seen the comment, but still, I'm not sure that I completely understand the purpose of `initial_realtime_reload_done`.

I've tried to me more explicit in the comment. 
Is it more understandable ?

The thing is that we may be disconnected from rabbitmq at any time :

-  in the middle of the initial realtime reload, and thus the reload  should be performed again
- or after the initial realtime reload was successfully done, and it is useless to perform it again

**Pull request merged**